src/js/MqttHandler.js

```diff
@@ -44,7 +44,7 @@ export default class MqttHandler extends Component {
         return;
       }
 
-      await this.iov.destroy();
+      this.iov.destroy();
```

Comment:
We should put a comment here explaining that iov.destroy() returns a promise and that we are purposefully not doing anything with it.